src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebReportRouteConfig.kt

```diff
@@ -37,6 +38,12 @@ object WebReportRouteConfig : RouteConfig
             WebEndpoint(
                     "/run-report/",
                     ReportController::class, "getRunReport")
-                    .secure(runReports)
+                    .secure(runReports),
+            WebEndpoint(
```

Comment:
Won't this clash with the publishReports method to get the page?

Reply:
No because it's a POST

Reply:
The endpoints won't clash because they've got different verbs, but aren't they both targeting a controller method called publishReports, which has different implementations in the two branches? 
https://github.com/vimc/orderly-web/pull/250/files#diff-02830e653e5e5fb2b4870a4cdb2dacf9R93
https://github.com/vimc/orderly-web/pull/248/files#diff-02830e653e5e5fb2b4870a4cdb2dacf9R64

Reply:
Ah sorry, misunderstood. Yes! I'll change the other one to `getPublishReports` which is consistent with your `getRunReports` method for that page.

Reply:
Thanks for pointing that out, fixed here https://github.com/vimc/orderly-web/pull/248/commits/5177805ae3f7eb7de5dff4b0f9db99f204253b73